index.js

```diff
@@ -4,31 +4,41 @@ const {Client} = require('discord.js');
 const Bot = require('./src/Bot.js');
 
 const client = new Client();
-const bot = new Bot({}, 10);
+const bot = new Bot(10);
 
 client.on('ready', () => {
   console.log('READY');
 });
 
 client.on('message', message => {
-  bot.notify(message.channel.id, new Date());
+  const {channel} = message;
+
+  bot.notify(channel.id, new Date());
 
   switch(message.content){
 
     case '!autobump':
-      bot.configure(message.channel.id, function (msg) {
-          message.channel.send(msg);
+      channel.send('Autobumping');
+      bot.configure(channel.id, () =>{
+          const {lastMessage} = channel;
+          // console.log('message', lastMessage.author, client.user);
+          if(lastMessage && lastMessage.author === client.user){
+            lastMessage.delete();
```

Comment:
Should it not do this when sending a message? (rather than when being configured)

Reply:
This is within the configure callback :D